subsys/usb/class/audio/audio.c

```diff
@@ -0,0 +1,215 @@
+#include <kernel.h>
+#include <usb/usb_common.h>
+#include <usb/usb_device.h>
+#include <usb_descriptor.h>
+#include <usb/class/usb_audio.h>
+
+#include <misc/byteorder.h>
+
+/*TODO: Kconfig options for below defines */
```

Comment:
Just a general note that ISO-specific HW information could be added to device tree.